docs/x509/reference.rst

```diff
@@ -1112,6 +1124,12 @@ X.509 CSR (Certificate Signing Request) Builder Object
         <NameAttribute(oid=<ObjectIdentifier(oid=2.5.4.10, name=organizationName)>, value=u'Test Certificates 2011')>
         <NameAttribute(oid=<ObjectIdentifier(oid=2.5.4.3, name=commonName)>, value=u'Good CA')>
 
+    .. attribute:: rdns
```

Comment:
Now that `Name` is an iterable of `RelativeDistinguishedName` objects does it make sense to keep this attribute?